.snapcraft/resources/restoredb

```diff
@@ -0,0 +1,72 @@
+#! /bin/bash
+
+if [[ ${EUID} != 0 ]]
+then
+    echo "[-] This task must be run with 'sudo'."
+    exit
+fi
+
+backup_file=${1}
+if [[ ! -f ${backup_file} ]]
+then
+    echo "[-] Usage: snap run rocketchat-server.rcrestore ${SNAP_COMMON}/backup_file.tgz"
```

Comment:
@geekgonecrazy seems I overlooked this when I renamed the scripts.
Change:
echo "[-] Usage: snap run rocketchat-server.~~rcrestore~~ ${SNAP_COMMON}/backup_file.tgz"
To:
echo "[-] Usage: snap run rocketchat-server.**restoredb** ${SNAP_COMMON}/backup_file.tgz"